cranelift/isle/veri/veri_engine/examples/load/load_conditional.isle

```diff
@@ -0,0 +1,25 @@
+;; Instruction formats.
+(type MInst (enum))
+
+;; Conditional with 2 loads spec
+(form
+  lhs_form
+  ((args Bool (bv  8) (bv  8)) (ret (bv  8)) (canon (bv  8)))
+  ((args Bool (bv 16) (bv 16)) (ret (bv 16)) (canon (bv 16)))
+  ((args Bool (bv 32) (bv 32)) (ret (bv 32)) (canon (bv 32)))
+  ((args Bool (bv 64) (bv 64)) (ret (bv 64)) (canon (bv 64)))
+)
+(spec (lhs cond x y) 
+  (provide (= result (if cond (load #x0000 (widthof x) x) (load #x0000 (widthof y) y)))))
```

Comment:
Now that we've gone back to the "one load per side" logic in the solver, you'll need to update this test (and the `switch` test) such that the `if` and the `switch` happen *inside* the `load`. That is: you should have one call to `load`, and each argument should have an `(if... )` or a `(switch...)` that specify the different arguments in the different cases.